### ISSUE TITLE
[HL2: EP1 & EP2 (GENERAL)] explosion_turret_break missing (due to hl2_episodic == 1)

### ISSUE THREAD

**Issue by BreakinBenny** [ValveSoftware/Source-1-Games#3762]
Because of how the code's set up, [https://github.com/ValveSoftware/source-sdk-2013/blob/master/sp/src/game/server/hl2/npc_turret_floor.cpp **npc_turret_floor** tries to precache the **explosion_turret_break** particle if **hl2_episodic** is active](url). Since these three games were put on the same engine build, this problem can come up if you're playing EP1 where the floor turrets have only one appearance(?)

Not to mention, the PCF file where **explosion_turret_break** and **explosion_turret_fizzle** are found in is located in EP2's game files. This could be moved to EP1's and the data added to its particles_manifest.txt

![20211215211137_1](https://user-images.githubusercontent.com/4566768/146409838-b0b6fbc6-dce4-4401-b813-6cccb1654cab.jpg)

**Comment by SC1040-TS2** [comment]
> Because of how the code's set up, [**npc_turret_floor** tries to precache the **explosion_turret_break** particle if **hl2_episodic** is active](https://github.com/ValveSoftware/source-sdk-2013/blob/master/sp/src/game/server/hl2/npc_turret_floor.cpp). Since these three games were put on the same engine build, this problem can come up if you're playing EP1 where the floor turrets have only one appearance(?)
> 
> Not to mention, the PCF file where **explosion_turret_break** and **explosion_turret_fizzle** are found in is located in EP2's game files. This could be moved to HL2's directory and the data added to its particles_manifest.txt, with the drawback of also adding the Combine Autogun's particles to the predecessors, of course.
> 
> ![20211215211137_1](https://user-images.githubusercontent.com/4566768/146409838-b0b6fbc6-dce4-4401-b813-6cccb1654cab.jpg)

Greetings.

While a bit of an odd interjection, the mention of particle precaching and trying to fix it through backporting the asset and adding things to the manifest files to reflect it reminded me of this prior thread I had started, https://github.com/ValveSoftware/Source-1-Games/issues/3615 , where I had sought to clean up the games' particle manifests to account for files still left in and otherwise.

However, the topic of this thread may mean that more work must be done to unify HL2, EP1, and EP2's asset repositories, as the engine is or may be failing to consistently parse and load things from other .VPK locations simultaneously as one would expect Source 2013 to do in most circumstances.

**Comment by BreakinBenny** [comment]
> > Replying to [#3762 (comment)](https://github.com/ValveSoftware/Source-1-Games/issues/3762#issue-1082412666)

Yes sir, I can see that work needs doing. I was also gonna mention that the unification of HL2's, EP1's, EP2's and even Lost Coast's assets would practically make good use of being merged somehow, so it's like one continuous game or campaign much like if you marathoned through Doom's and Quake's episodes (mainly the first Quake, as you get the maps as part of the "registered" product while Doom doesn't unless you play on particular non-PC ports like the SNES one)... If Lost Coast could be made into a "bonus map" like in the way Portal's bonus map system works.
